StreamChatClient_v3/Workers/ChannelUpdater.swift

```diff
@@ -19,4 +18,15 @@ class ChannelUpdater<ExtraData: ExtraDataTypes>: Worker {
             }
         }
     }
+
+    func muteChannel(cid: ChannelId, mute: Bool, completion: ((Error?) -> Void)? = nil) {
+        apiClient.request(endpoint: .muteChannel(cid: cid, mute: mute)) {
+            switch $0 {
+            case .success:
+                completion?(nil)
+            case let .failure(error):
+                completion?(error)
+            }
+        }
+    }
```

Comment:
This also needs to be tested. I'm currently in the process of updating `ChannelUpdater` tests so we can skip this for now.